python/cudf_polars/cudf_polars/experimental/parallel.py

```diff
@@ -14,7 +14,15 @@
 import cudf_polars.experimental.join
 import cudf_polars.experimental.select
 import cudf_polars.experimental.shuffle  # noqa: F401
-from cudf_polars.dsl.ir import IR, Cache, Filter, HStack, Projection, Select, Union
+from cudf_polars.dsl.ir import (
+    IR,
+    Cache,
+    Filter,
+    HStack,
+    Projection,
+    Select,
+    Union,
+)
```

Comment:
```suggestion
from cudf_polars.dsl.ir import IR, Cache, Filter, HStack, Projection, Select, Union
```
No need for this change I think.